host/config.go

```diff
@@ -17,6 +17,7 @@ var defaultConfig = Config{
 	Websocket:                          false,
 	BootNodesReachabilityCheckInterval: 1 * time.Minute,
 	MustReachBootNodes:                 defaultMustReachBootNodes,
+	EnableP2PRelay:                     true,
```

Comment:
Interesting - do we want nodes to be relays by default? @dmikey 

Potentially we could say only head nodes are relays by default, and in main.go say `host.WithEnableP2PRelay(role == blockless.HeadNode)`.

However I do think having more relays would make the network more decentralized and resilient, if there's no drawbacks. I'm cool with the current option of having it on, just bringing it up for discussion.

Reply:
I think relays make sense. Let's start with head nodes, and maybe also `dialback` flag as a required option when enabling relay. More likely the operator would have tested / knows about inbound routing which would still be needed for the relay to be effective. This will fix our problem with formation dialing as well "could not reach peer" we've been seeing in the wild.

Reply:
addressed p2p relay default option: https://github.com/blocklessnetwork/b7s/pull/168/commits/8ebb63bc3c5c121fae7203d916409918f38abc52

unsure about which dialback config we are talking about here though.

Reply:
there is an optional --dialback-address --dialback-port, these are used for advertising the explicit return address to the node, though I don't think I like this after thinking it through. we'll keep it independent.